disks-and-volumes.md

```diff
@@ -0,0 +1,315 @@
+# Volumes and Disks
+
```

Comment:
Maybe a few introductional words to state that th eterm disk coins the guest facing side, whereas volumes specifies the kube source side.

Reply:
done